script/test/fixtures/configmap-pod/output-os.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    io.kompose.service: redis
+  name: redis
+spec:
+  ports:
+    - name: "6379"
+      port: 6379
+      targetPort: 6379
+  selector:
+    io.kompose.service: redis
+  type: NodePort
+
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  labels:
+    io.kompose.network/configmap-pod-default: "true"
+    io.kompose.service: redis
+  name: redis
+spec:
+  containers:
+    - env:
+        - name: ALLOW_EMPTY_PASSWORD
+          valueFrom:
+            configMapKeyRef:
+              key: ALLOW_EMPTY_PASSWORD
+              name: foo-env
+        - name: BAR
+          valueFrom:
+            configMapKeyRef:
+              key: BAR
+              name: bar-env
+        - name: FOO
+          valueFrom:
+            configMapKeyRef:
+              key: FOO
+              name: bar-env
```

Comment:
@realgam3 I see here that you are referencing a config map which is not generated. I think that we should handle it in the `openshift.go` file.

Reply:
@AhmedGrati Great catch! I'm not sure how this one slipped past me. :S
Fixed and pushed....